### ISSUE TITLE
[Bug] App crashes on startup if system language is set to Marathi (मराठी)

### ISSUE THREAD

**Issue by quanta-kt** [ankidroid/Anki-Android#10545]
###### Reproduction Steps

1. Set your system language to Marathi
2. Start the app

###### Workaround

1. Temporarily set system language to English
2. Open Anki droid and set it's language to English from in-app settings
3. Set system language back to what you use

This way the app works, but only if you are willing to use it in English. Well, that's at-least better than completely an unusable app.

###### Expected Result

The app to start normally

###### Actual Result

Crashes on start-up

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".
```
AnkiDroid Version = 2.15.6

Android Version = 11

Manufacturer = OPPO

Model = CPH1937

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 11; CPH1937 Build/RKQ1.200903.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/99.0.4844.58 Mobile Safari/537.36

ACRA UUID = 20958df6-e5ff-44d7-bcbd-2657cc5434d5

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [ ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by rajath02** [comment]
If no one is assigned currently, I can work on this

**Comment by AditiKamath** [comment]
I would like to work on this

**Comment by krmanik** [comment]
Hi, The issue is in the translation. It needs to update/fix the string here [crowdin.com](https://crowdin.com/translate/ankidroid/7290/en-mr?filter=basic&value=0#q=one)

The other fix is to check the string for format error using lint.

![image](https://user-images.githubusercontent.com/12841290/159148246-51da9b71-436e-437b-8c04-f7a9f83944de.png)

**Comment by quanta-kt** [comment]
Oh that is terrible. I have been peeking around the Marathi and Hindi translations and it seems that people who volunteered for those languages have simply copy pasted machine translations. I have submitted a few translations which I noticed to be exceptionally odd.

**Comment by mikehardy** [comment]
Thank you for helping out with the translations. That *should* fix the crash, but unfortunately there is nothing we can do to fix version 2.15.6 as we are blocked from releasing an update to the app until the huge work for #5304 is done. In the meantime, the next time I release an alpha - assuming you have fixed the translation - the language files will be in sync and you should be able to run normally with the new version from here: https://github.com/ankidroid/Anki-Android/releases

I'll start that process now since I haven't done a release in a few days, should be a new version up with fresh translations in a few hours. If it doesn't work, we can reopen

**Comment by david-allison** [comment]
We should lint this error
